tests/integration_tests/test_audio_feature.py

```diff
@@ -7,7 +7,7 @@
 BATCH_SIZE = 2
 SEQ_SIZE = 20
 AUDIO_W_SIZE = 16
-DEFAULT_FC_SIZE = 256
+DEFAULT_OUTPUT_SIZE = 256
```

Comment:
this default is likely too big, let's decrease it for faster tests

Reply:
Done